services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/GetItemEnhancedResponse.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.enhanced.dynamodb.model;
+
+import java.util.Objects;
+import software.amazon.awssdk.annotations.NotThreadSafe;
+import software.amazon.awssdk.annotations.SdkPublicApi;
+import software.amazon.awssdk.annotations.ThreadSafe;
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbAsyncTable;
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbTable;
+import software.amazon.awssdk.services.dynamodb.model.ConsumedCapacity;
+import software.amazon.awssdk.services.dynamodb.model.GetItemResponse;
+
+/**
+ * Defines the elements returned by DynamoDB from a {@code GetItem} operation, such as
+ * {@link DynamoDbTable#getItemWithResponse(GetItemEnhancedRequest)} and
+ * {@link DynamoDbAsyncTable#getItemWithResponse(GetItemEnhancedRequest)}.
+ *
+ * @param <T> The type of the item.
+ */
+@SdkPublicApi
+@ThreadSafe
+public final class GetItemEnhancedResponse<T> {
+    private final T attributes;
+    private final ConsumedCapacity consumedCapacity;
+
+    private GetItemEnhancedResponse(GetItemEnhancedResponse.Builder<T> builder) {
+        this.attributes = builder.attributes;
+        this.consumedCapacity = builder.consumedCapacity;
+    }
+
+    /**
+     * The attribute values values returned by {@link GetItem} operation.
```

Comment:
extra "values"
should be `{@code GetItem}` instead of `@link `

Reply:
Good eye.